packages/frontend/src/notebook/notebook_editor.tsx

```diff
@@ -232,19 +232,28 @@ export function NotebookEditor<T>(props: {
                             hasFocused: () => {
                                 setActiveCell(i());
                             },
-                            moveCellUp: () => {
+                         moveCellUp: () => {
```

Comment:
Looks like a Git issue caused from old code related to #19. This file should have no changes in this PR.